### PR TITLE
Add checks for empty content uris in `Cesium3DTile`

### LINKED ISSUE #7263: `Cesium3DTileset` loads tileset recursively on empty content URL
While writing 3D Tiles generator I've found out that an empty URL in the Tile's content (i.e. `"content": { "url": "" }`) causes cycle in the tile tree structure in the Cesuim API.

The core cause is because content's full URL is then the original tileset's URL and fetched JSON is treated as an external tileset. If there is more than one tile with an empty URL it is an equivalent of a fork bomb: the browser slows down terribly, endlessly loading _external_ tilesets and actual tile data over and over again.

Propper handling should be to report invalid tile and maybe skip it.

### PR DESCRIPTION
**Pull request by j9liu**
Fixes #7263. As discussed in the issue, if a tile contains an empty content uri, it will print a warning to the console and treat itself like a regular empty tile.

I'm not sure I handled all processes involved with the `"3DTILES_multiple_contents"` extension properly -- please let me know if I need to make changes.

cc @ebogo1 or @lilleyse

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @j9liu!
* :heavy_check_mark: Signed CLA found.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

**Comment by lilleyse** [comment]
I tagged @ptrgags to review when he's back. He's most familiar with this area of the code.

File: Source/Scene/Cesium3DTile.js [hunk 1]
```diff
@@ -220,13 +221,23 @@ function Cesium3DTile(tileset, baseResource, header, parent) {
...
-    contentState = Cesium3DTileContentState.UNLOADED;
-    contentResource = baseResource.getDerivedResource({
-      url: contentHeaderUri,
-    });
-    serverKey = RequestScheduler.getServerKey(
-      contentResource.getUrlComponent()
-    );
+
+    if (contentHeaderUri === "") {
+      console.log(
```
Comment (ebogo1) [inline Source/Scene/Cesium3DTile.js:226]: should this use `oneTimeWarning` instead? there's an eslint error because the import is unused.

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @j9liu! 

No one has commented on this pull request in 90 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 90 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @j9liu! 

No one has commented on this pull request in 90 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 90 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by ggetz** [comment]
@j9liu To keep things tidy, I'm going to close this PR. However if you have any updates, feel free to re-open. Thanks!

**Pull request closed without merging**